grpc-cpp/plan.sh

```diff
@@ -0,0 +1,227 @@
+    -DProtobuf_PROTOC_LIBRARY="${PROTOBUF_PATH}/lib/libprotoc.so" \
+    -DgRPC_SSL_PROVIDER="package" \
+    -G "Unix Makefiles" \
+    ../..
+  make -j"$(nproc --ignore=1)"
+  popd || exit 1
+}
+
+do_check() {
+  testdir="${BUILDDIR}/testenv"
+  fulltestdir_path="$(realpath "$testdir")"
+
+  # clean up existing testenv
+  rm -rf "${testdir}"
+
+  # setup env
+  virtualenv "${testdir}"
+  source "${testdir}/bin/activate"
+
+  # add py deps
+  pip install six
+  pip install pyyaml
+  pip install twisted
+  pip install httplib2
+
+  # fix the interpreters without adding more runtime deps
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/python_utils/antagonist.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/start_port_server.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/core/http/test_server.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/core/bad_client/gen_build_yaml.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/core/bad_ssl/gen_build_yaml.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/core/end2end/fuzzers/generate_client_examples_of_bad_closing_streams.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/cpp/naming/resolver_component_tests_runner.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/cpp/naming/utils/tcp_connect.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/cpp/naming/utils/dns_server.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/cpp/naming/utils/dns_resolver.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i test/cpp/naming/utils/run_dns_server_for_lb_interop_tests.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/sanity/check_port_platform.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/sanity/check_version.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/sanity/core_banned_functions.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/sanity/check_deprecated_grpc++.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/performance/patch_scenario_results_schema.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/task_runner.py
+  sed -e "s#/usr/bin/env python.*#${fulltestdir_path}/bin/python2#" -i tools/run_tests/run_tests.py
+
+  # fix paths to tests
+  sed -e "s#\-\-generated_file_path=gens#\-\-generated_file_path=${BUILDDIR}/gens#g" \
+      -i tools/run_tests/generated/tests.json
+
+  # increase file limits
+  ulimit -n 1000000
+
+  # run tests
+  if ! ifconfg | grep -v "inet6"; then
```

Comment:
Typo?  `ifconfig`

Reply:
Oh snap!  Yes you are right it should be ifconfig!